app/containers/News/News.jsx

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react'
+import moment from 'moment'
+
+import Panel from '../../components/Panel'
+import HeaderBar from '../../components/HeaderBar'
+import styles from './News.scss'
+import Loader from '../../components/Loader'
+
+const electron = require('electron').remote
+
+type Props = {
+  feed: {
+    items: Array<{
+      content: string,
+      creator: string,
+      isoDate: Date,
+      title: string,
+      link: string,
+    }>,
+  },
+  loading: boolean,
+  networkId: string,
+  net: string,
+}
+
+export default class News extends React.Component<Props> {
+  render() {
+    return (
+      <div className={styles.newsContainer}>
+        <HeaderBar
+          networkId={this.props.networkId}
+          net={this.props.net}
+          label="News"
+        />
+        <Panel className={styles.newsPanel}>
+          {this.props.loading ? <Loader /> : this.parseItems()}
+        </Panel>
+      </div>
+    )
+  }
+
+  parseItems() {
+    const { items } = this.props.feed
+    const imgTagRegex = new RegExp('<s*img[^>]*>(.*?)')
+
+    const imageHrefFromImgTags = img =>
```

Comment:
Getting the image href is pretty ugly like this but it was hard for me to think of anything cleaner/simpler as there is no image key on the available parsed XML